lua/nvim-treesitter/parsers.lua

```diff
@@ -1081,6 +1081,18 @@ list.v = {
   maintainers = { "@tami5" },
 }
 
+list.tiger = {
+  install_info = {
+    url = "https://github.com/ambroisie/tree-sitter-tiger",
+    files = { "src/parser.c", "src/scanner.c" },
```

Comment:
you're using a custom scanner. It SEGFAULTs when I'm trying to run the fuzzer 
```
INFO: Running with entropic power schedule (0xFF, 100).
INFO: Seed: 1225235103
INFO: Loaded 1 modules   (16464 inline 8-bit counters): 16464 [0x560154b0be50, 0x560154b0fea0), 
INFO: Loaded 1 PC tables (16464 PCs): 16464 [0x560154b0fea0,0x560154b503a0), 
INFO: -max_len is not provided; libFuzzer will not generate inputs larger than 4096 bytes
tiger_fuzzer: test/fuzz/fuzzer.cc:45: int LLVMFuzzerTestOneInput(const uint8_t *, size_t): Assertion `language_ok' failed.
==12244== ERROR: libFuzzer: deadly signal
    #0 0x5601548aedb1 in __sanitizer_print_stack_trace (/home/stephan/tree-sitter/out/tiger_fuzzer+0x19ddb1) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #1 0x56015481e9d8 in fuzzer::PrintStackTrace() (/home/stephan/tree-sitter/out/tiger_fuzzer+0x10d9d8) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #2 0x560154804c73 in fuzzer::Fuzzer::CrashCallback() (/home/stephan/tree-sitter/out/tiger_fuzzer+0xf3c73) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #3 0x7f665c8ed51f  (/lib/x86_64-linux-gnu/libc.so.6+0x4251f) (BuildId: 89c3cb85f9e55046776471fed05ec441581d1969)
    #4 0x7f665c941a7b in __pthread_kill_implementation nptl/./nptl/pthread_kill.c:43:17
    #5 0x7f665c941a7b in __pthread_kill_internal nptl/./nptl/pthread_kill.c:78:10
    #6 0x7f665c941a7b in pthread_kill nptl/./nptl/pthread_kill.c:89:10
    #7 0x7f665c8ed475 in gsignal signal/../sysdeps/posix/raise.c:26:13
    #8 0x7f665c8d37f2 in abort stdlib/./stdlib/abort.c:79:7
    #9 0x7f665c8d371a in __assert_fail_base assert/./assert/assert.c:92:3
    #10 0x7f665c8e4e95 in __assert_fail assert/./assert/assert.c:101:3
    #11 0x5601548e3749 in LLVMFuzzerTestOneInput (/home/stephan/tree-sitter/out/tiger_fuzzer+0x1d2749) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #12 0x560154806212 in fuzzer::Fuzzer::ExecuteCallback(unsigned char const*, unsigned long) (/home/stephan/tree-sitter/out/tiger_fuzzer+0xf5212) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #13 0x560154807470 in fuzzer::Fuzzer::ReadAndExecuteSeedCorpora(std::vector<fuzzer::SizedFile, std::allocator<fuzzer::SizedFile>>&) (/home/stephan/tree-sitter/out/tiger_fuzzer+0xf6470) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #14 0x560154807ac2 in fuzzer::Fuzzer::Loop(std::vector<fuzzer::SizedFile, std::allocator<fuzzer::SizedFile>>&) (/home/stephan/tree-sitter/out/tiger_fuzzer+0xf6ac2) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #15 0x5601547f5f02 in fuzzer::FuzzerDriver(int*, char***, int (*)(unsigned char const*, unsigned long)) (/home/stephan/tree-sitter/out/tiger_fuzzer+0xe4f02) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #16 0x56015481f2c2 in main (/home/stephan/tree-sitter/out/tiger_fuzzer+0x10e2c2) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)
    #17 0x7f665c8d4d8f in __libc_start_call_main csu/../sysdeps/nptl/libc_start_call_main.h:58:16
    #18 0x7f665c8d4e3f in __libc_start_main csu/../csu/libc-start.c:392:3
    #19 0x5601547eaac4 in _start (/home/stephan/tree-sitter/out/tiger_fuzzer+0xd9ac4) (BuildId: 0e91adb39e4e209364e5584b96aae88be1bfa2e7)

NOTE: libFuzzer has rudimentary signal handlers.
      Combine libFuzzer with AddressSanitizer or similar for better crash reports.
SUMMARY: libFuzzer: deadly signal
MS: 0 ; base unit: 0000000000000000000000000000000000000000


artifact_prefix='./'; Test unit written to ./crash-da39a3ee5e6b4b0d3255bfef95601890afd80709
Base64: 

```
Instructions here: https://github.com/ikatyang/tree-sitter-markdown/issues/14

Reply:
just saw that it failed to load the language. let me check...

Reply:
sorry, was wrong alarm. No problems with the scanner!